src/utils/getIconHtml.ts

```diff
@@ -0,0 +1,32 @@
+import { library, icon, IconDefinition, Icon, IconName } from '@fortawesome/fontawesome-svg-core';
+import * as solidIcons from '@fortawesome/free-solid-svg-icons';
+
+export function getIconHtmlFromIconLookup(iconName: IconName | string): string | null {
```

Comment:
make 2 files each with the name of the function so it's easy to find

Reply:
done